index.html

```diff
@@ -0,0 +1,75 @@
+ <!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" href="normalize.css">
+	<link rel="stylesheet" href="styles.css">
```

Comment:
best practice is to move your stylesheet to be last one defined (under your title ) - also it needs type="text/css"